src/common/packager.ts

```diff
@@ -43,7 +45,7 @@ export class Packager {
                             let args = ["--port", Packager.PORT];
                             let childEnvForDebugging = Object.assign({}, process.env, { REACT_DEBUGGER: "echo A debugger is not needed: " });
 
-                            Log.logMessage("Starting Packager", outputChannel);
+                            Log.logMessage("Starting Packager");
```

Comment:
What is the default logging behavior inside the extension process? Prior to this change, all these messages go to whatever outputChannel is passed in (which I believe is one created specifically for the packager) . Now that you are removing the outputChannel here, the "starting packager" message will go to the default location, while the process output will go to the passed-in channel. Is that a conscious change?